ephios/static/ephios/scss/ephios_custom.scss

```diff
@@ -266,3 +266,12 @@ a.badge {
 .notification-content ul {
   padding-bottom: 1rem;
 }
+
+.day-calendar-container {
+  display: grid;
+}
+
+.shift_box:target {
+  border-color: #9ecaed;
+  box-shadow: 0 0 10px #9ecaed;
```

Comment:
I really like the feature, I'm not 100% comfortable with the color yet, but I can't think of a better way. I'm guessing this is derived from the primary color. Can you use the sass variable here?

Reply:
It was based of default browser behaviour, but I've changed it to `$info`. Is that ok for you?

<img width="1364" alt="Screenshot 2024-03-13 at 14 51 56" src="https://github.com/ephios-dev/ephios/assets/2546622/2a8b919e-e7d8-4e82-8637-fec6efe5f7d3">

Reply:
I think it looks more in palette. :)
How does $primary look?

Reply:
<img width="681" alt="Screenshot 2024-03-13 at 15 29 00" src="https://github.com/ephios-dev/ephios/assets/2546622/c4ac92a5-a09d-471e-a5ee-38505ce6d766">

I think `$primary` is too far away from the default browser thing to be easily recognisable by most users as the "focus" color.

Reply:
Hmm yeah I find it too dark. I like

```scss
$shift-box-target: lighten($primary, 20%);
.shift_box:target {
  border-color: $shift-box-target;
  box-shadow: 0 0 10px $shift-box-target;
}
```

In the past we had the info color reserved for administrative and planning actions, that's why I feel like avoiding it here.

Reply:
lgtm